src/desktop/apps/isomorphic-relay-example/routes.js

```diff
@@ -0,0 +1,102 @@
+import React from 'react'
+import { graphql } from 'react-relay'
+import { Layout } from './components/Layout'
+import { HomeRoute } from './routes/home/HomeRoute'
+import { ArtistRoute } from './routes/artist/ArtistRoute'
+import { AuctionRoute } from './routes/auction/AuctionRoute'
+import { ReactLoadableClientRoute } from './routes/react-loadable/ReactLoadableClientRoute'
+import { ReactLoadableServerRoute } from './routes/react-loadable/ReactLoadableServerRoute'
+
+export const routes = [
+  {
+    Component: Layout,
+    path: '/isomorphic-relay-example',
+    children: [
+      {
+        path: '/',
+        Component: HomeRoute,
+      },
+      {
+        path: '/about',
+        Component: () => <div>About page!</div>,
+      },
+      {
+        path: '/artsy/:artistID',
+        prepareVariables: params => ({ artistID: 'pablo-picasso' }),
+        getComponent: ({ artworks }) => {
+          return import('@artsy/reaction/dist/Components/ArtworkGrid').then(
```

Comment:
@damassi 👏 That's awesome. Did you have to add a babel plugin to support dynamic imports? Also why not use `loadable-components` here?

Reply:
Ah yah this was deliberate -- I wanted to show what a vanilla import example looks like

Reply:
For SSR I had to use Airbnb's [`plugin-dynamic-import-node`](https://github.com/airbnb/babel-plugin-dynamic-import-node) -- this was a bit tricky to figure out becuase if its in the babel configuration directly then webpack won't do bundle splitting, so it has to be imported via `require('@babel/register', { plugins: [...] })`